tests/fixture_plugins/demultiplex_fixtures/name_fixtures.py

```diff
@@ -1,27 +1,23 @@
+"""Module for demultiplex fixtures returning strings."""
+
 import pytest
 
 
 @pytest.fixture
 def flow_cell_name() -> str:
-    """Return flow cell name."""
+    """Return flow cell name (novaseq pre 1.5 I think)."""
```

Comment:
```suggestion
    """Return flow cell name (presumably novaseq pre 1.5)."""
```

Reply:
sorry, that docstring was not supposed to be there, it was just for my reference. I will remove it